app/tests/src/index.js

```diff
@@ -1,28 +1,31 @@
 describe('AdaLite Test Suite', () => {
   require('./common/setup-test-config')
 
-  describe('CBOR', () => {
-    require('./cbor')
-  })
-  describe('Mnemonic Crypto Provider', () => {
-    require('./cardano-wallet-secret-crypto-provider')
-  })
-  describe('Address Manager', () => {
-    require('./address-manager')
-  })
-  describe('Blockchain Explorer', () => {
-    require('./blockchain-explorer')
-  })
-  describe('Cardano Wallet', () => {
-    require('./cardano-wallet')
-  })
-  describe('Import/Export Wallet as JSON', () => {
-    require('./keypass-json')
-  })
-  describe('Actions', () => {
-    require('./actions/actions')
-  })
-  describe('Shelley testnet', () => {
-    require('./shelley')
+  // describe('CBOR', () => {
```

Comment:
I would keep the cbor tests, they seem to be testing live code (CborIndefiniteLength array serialization)

Reply:
Of course, I am not planning on dropping any test right now, these were commented out just for development purposes, I ll make the removing the unused test into another PR..